versions.props

```diff
@@ -26,3 +25,6 @@ org.mockito:mockito-core = 2.27.0
 org.mpierce.metrics.reservoir:* = 1.1.2
 org.openjdk.jmh:* = 1.21
 org.slf4j:* = 1.7.12
+
+# dependency-upgrader:OFF
+io.dropwizard.metrics:metrics-core = 3.2.5
```

Comment:
Let's add an `# dependency-upgrader:ON` line after this, otherwise we'll add a new dependency at the bottom and wonder why it's not upgrading.